code/game/machinery/cryopod.dm

```diff
@@ -418,12 +418,18 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 	linked_ship = port
 	linked_ship.spawn_points += src
 
+/obj/machinery/cryopod/apply_effects_to_mob(mob/living/carbon/sleepyhead)
+	sleepyhead.SetSleeping(50)
+	sleepyhead.set_disgust(60)
+	sleepyhead.set_nutrition(150)
+	to_chat(sleepyhead, "<span class='bolddanger'>A wave of nausea comes over you, brought on by cryosleep...</span>")
+
 /obj/machinery/cryopod/poor
 	name = "low quality cryogenic freezer"
 	desc = "Keeps crew frozen in cryostasis until they are needed in order to cut down on supply usage. This one seems cheaply made."
 
-/obj/machinery/cryopod/poor/apply_effects_to_mob(mob/living/carbon/sleepyhead)
+/obj/machinery/cryopod/apply_effects_to_mob(mob/living/carbon/sleepyhead)
```

Comment:
I like it but this will hit everyone instead of just Poor "People"

Reply:
maybe some freezerburn ngl

Reply:
oh oops. copy paste wins again